email_api/server.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const app = express();
+
+var emailobj = require('./email');
```

Comment:
var emailobj is not following came case also it should be const

Reply:
resolved the camel case issue

Reply:
Go through other file comments as well and push as single change